contrib/caldav/README.md

```diff
@@ -49,7 +49,7 @@ How It Works
 ------------
 
 calcurse-caldav creates a so-called synchronization database at
-~/.calcurse/caldav/sync.db that always keeps a snapshot of the last time the
+`~/.calcurse/caldav/sync.db` that always keeps a snapshot of the last time the
```

Comment:
All the changes above do not seem to be related to this commit. They deserve a separate patch or could be squashed into the `README.md` patch.